src/plot/series/arc-series.js

```diff
@@ -130,15 +130,22 @@ class ArcSeries extends AbstractSeries {
     const opacityFunctor = this._getAttributeFunctor('opacity');
     const xFunctor = this._getAttributeFunctor('x');
     const yFunctor = this._getAttributeFunctor('y');
+    const focusable = this._getFocusable();
 
     const xTranslate = xFunctor(center);
     const yTranslate = yFunctor(center);
 
     return (
       <g className={`${predefinedClassName} ${className}`}
+        tabIndex = {focusable ? 0 : null}
```

Comment:
is there a way we could make like mixin for this big batch of props? it seems like every series has this now, so it would be easier to maintain if we were only injecting in a single place?